scapy/modules/nmap.py

```diff
@@ -124,12 +128,12 @@ def nmap_udppacket_sig(snd, rcv):
         res["ULEN"] = "%X" % rcv[UDPerror].len
         res["DAT"] = "E" if (
             isinstance(rcv[UDPerror].payload, NoPayload) or
-            raw(rcv[UDPerror].payload) == raw(snd[UDP].payload)
-        ) else "F"
+            raw(rcv[UDPerror].payload) == raw(snd[UDP].payload)) else "F"
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok